jenkins-jobs/nci/templates/publisher.xml.erb

```diff
@@ -68,6 +68,7 @@
       <flatten>true</flatten>
       <doNotFingerprintArtifacts>false</doNotFingerprintArtifacts>
     </hudson.plugins.copyartifact.CopyArtifact>
+<% if type == 'unstable' %>
```

Comment:
PublishJob should get all architectures it is meant to publish and then pack it into `attr_reader :architectures` and then inside the xml template you should `architectures.each do |arch|` and template the entire CopyArtifact block. The publisher mustn't know or care that extra_architectures are limited by type